test-apps/imodel-transformer/src/IModelHubUtils.ts

```diff
@@ -23,10 +23,10 @@ export namespace IModelHubUtils {
     await client.initialize({
       clientId: "imodeljs-electron-test",
       redirectUri: "http://localhost:3000/signin-callback",
-      scope: "openid email profile organization imodelhub context-registry-service:read-only reality-data:read product-settings-service projectwise-share urlps-third-party imodel-extension-service-api offline_access",
+      scope: "openid email profile organization itwinjs",
     });
     return new Promise<AccessToken>((resolve, reject) => {
-      NativeHost.onUserStateChanged.addListener((token) => {

```

Comment:
@calebmshafer Did you mean to leave this in? Right now ElectronAuthorizationBackend doesn't have an `onUserStateChanged` as of right now. The only way to get the token from the sign-in is from Native host's event.

Reply:
Ah nope, I accidentally committed this along with other changes that I had stashed. I'll revert this.

Reply:
Gotcha. I'm guessing this is where `ElectronAppAuthorization` (which currently doesn't exist) is coming from in full-stack-tests\core\src\frontend\_Setup.test.ts too then?

Reply:
Yup, fixing that now too. Should have a push in a minute or two.

Reply:
Pushed a fix for this. There's a couple tests failing now. Validating the test-apps again to make sure they're all working.